spec.html

```diff
@@ -0,0 +1,110 @@
+<!doctype html>
+<meta charset="utf8">
+<script src="ecmarkup.js"></script>
+<link rel="stylesheet" href="ecmarkup.css">
+<title>Module attributes</title>
+<pre class=metadata>
+  title: Module attributes
+  status: proposal
+  stage: 1
+  location: https://github.com/tc39/proposal-module-attributes
+  copyright: false
+  contributors: Sven Sauleau, Myles Borins, Daniel Ehrenberg, Daniel Clark
+</pre>
+<emu-intro id="intro">
+  <h1>Module attributes</h1>
+  <p>See <a href="https://github.com/tc39/proposal-module-attributes/blob/master/README.md">the explainer</a> for information.</p>
+</emu-intro>
+
+<emu-clause id="sec-syntax">
+  <h1>Syntax</h1>
+
+  <emu-grammar>
+    ImportDeclaration :
+      `import` ImportClause FromClause `;`
+      `import` ModuleSpecifier `;`
+      <ins>`import` ImportClause FromClause AsClause `;`</ins>
+      <ins>`import` ModuleSpecifier AsClause `;`</ins>
+
+    AsClause :
+      <ins>[no LineTerminator here] `as` StringLiteral</ins>
+
+    ImportCall[Yield, Await] :
+      `import` `(` AssignmentExpression[+In, ?Yield, ?Await] `)`
+      <ins>`import` `(` AssignmentExpression[+In, ?Yield, ?Await] `,` AssignmentExpression[+In, ?Yield, ?Await]`)`</ins>
+  </emu-grammar>
+</emu-clause>
+
+<emu-clause id="sec-semantics">
+  <h1>Semantics</h1>
+
+  <emu-note type="editor"><p>Many productions operating on grammar are the same whether or not an |AsClause|/second |ImportCall| parameter is included; the new parameter is ignored. In this section, only the semantically significant changes are included, and the PR to merge into the main specification would fill in the straightforward details.</p></emu-note>
+
+    <emu-clause id="sec-import-calls">
+      <h1>Import Calls</h1>
+
+      <emu-clause id="sec-import-call-runtime-semantics-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+
+        <emu-grammar>ImportCall : `import` `(` AssignmentExpression `)`</emu-grammar>
+        <emu-alg>
+          1. Let _referencingScriptOrModule_ be ! GetActiveScriptOrModule().
+          1. Let _argRef_ be the result of evaluating |AssignmentExpression|.
+          1. Let _specifier_ be ? GetValue(_argRef_).
+          1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+          1. Let _specifierString_ be ToString(_specifier_).
+          1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
+          1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, _specifierString_, <ins>*undefined*,</ins> _promiseCapability_).
+          1. Return _promiseCapability_.[[Promise]].
+        </emu-alg>
+
+        <ins><emu-grammar>ImportCall : `import` `(` AssignmentExpression, AssignmentExpression `)`</emu-grammar></ins>
```

Comment:
`import()` failed to allow a trailing comma in its initial version, but perhaps we could include that if we're making changes?

Reply:
Oh, sure, makes sense to me. Will update.

Reply:
Done